zbstudio.appdata.xml.patch

```diff
@@ -1,5 +1,5 @@
---- ZeroBraneStudio-linux/app/zbstudio/res/zbstudio.appdata.xml	2023-02-05 08:00:08.019262840 +0200
-+++ ZeroBraneStudio-linux/app/zbstudio/res/zbstudio.appdata.xml	2023-02-06 17:30:23.170086637 +0200
+--- a/zbstudio/res/zbstudio.appdata.xml	2023-02-05 08:00:08.019262840 +0200
++++ b/zbstudio/res/zbstudio.appdata.xml	2023-02-06 17:30:23.170086637 +0200
 @@ -1,6 +1,6 @@
  <?xml version="1.0" encoding="UTF-8"?>
  <application>
```

Comment:
Has to start with component tag https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#header